bin/query-db-and-email

```diff
@@ -573,7 +600,7 @@ function getCredentialsFromUrl (url) {
 }
 
 function getResponse (apiPath, qs) {
-  const requestOptions = {qs};
+  const requestOptions = {qs, qsStringifyOptions: {arrayFormat: 'brackets'}};
```

Comment:
This update isn’t critical for the current use case, but is coming along for the ride since I needed it for testing. We weren’t serializing the query quite correctly in cases when multiple tags were involved.